index.html

```diff
@@ -2,11 +2,48 @@
 <html>
   <head>
     <meta charset="utf-8">
-    <title>Change the title</title>
+    <title>Gaming Screen</title>
+
+    <link rel="stylesheet" href="style.css">
     <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.6.3/css/font-awesome.css">
+    <link href='https://fonts.googleapis.com/css?family=Yanone+Kaffeesatz:700' rel='stylesheet' type='text/css'>
   </head>
   <body>
+    <div class="content">
+
+        <div class="leftside">
+          <img class="avatar-img" src="http://www.mmo-town.com/wp-content/uploads/2016/03/winston-profile-icon.png" alt="Placeholder">
+            <div class="healthstuff">
+              <p class="healthscore"><mark class="first500">500</mark><mark class="healthscoreSL">/</mark>500</p>
+              <i class="fa fa-heartbeat" aria-hidden="true"></i>
+              <i class="fa fa-heartbeat" aria-hidden="true"></i>
+              <i class="fa fa-heartbeat" aria-hidden="true"></i>
+              <i class="fa fa-heartbeat" aria-hidden="true"></i>
+              <i class="fa fa-heartbeat" aria-hidden="true"></i>
+            </div>
+        </div>
+
+        <div class="middleside">
```

Comment:
For the middle items, I would break the smiley face image away from the "Eliminated" text and then position each group `fixed`

Once you have that, you can use `transform: translate...` to get things perfectly centered https://rtablada.gitbooks.io/summer-2016/content/week-1/day-3/transform.html